pkg/images/enricher/enricher_impl.go

```diff
@@ -581,6 +582,13 @@ func (e *enricherImpl) enrichWithScan(ctx context.Context, enrichmentContext Enr
 				})
 			}
 			errorList.AddError(err)
+
+			if features.ScannerV4.Enabled() && scanner.GetScanner().Type() == types.ScannerV4 {
+				// Do not try to scan with additional scanners if Scanner V4 enabled and fails to scan an image.
+				// Per sorting logic in pkg/scanners this would result in the Clairify scanners being skipping.
```

Comment:
Can you maybe add a comment here to point to where `GetAll` is defined and in the `GetAll` to let devs know changing the order will affect this?